src/auth/auth.service.spec.ts

```diff
@@ -0,0 +1,141 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import { AuthService } from './auth.service';
+import { User } from 'src/types/user';
+import { UsersService } from '../users/users.service';
+import { HttpException, BadRequestException, HttpStatus } from '@nestjs/common';
+import { GoogleDTO } from 'src/auth/auth.dto';
+import * as bcrypt from 'bcrypt';
+import { JwtService } from '@nestjs/jwt';
+
+const SALT_ROUNDS = 10;
+
+describe('AuthService', () => {
+  let authService: AuthService;
+  let userService: UsersService;
+  let jwtService: JwtService;
+
+  beforeEach(async () => {
+    const module: TestingModule = await Test.createTestingModule({
```

Comment:
Zamiast podnosić testing module można to potraktować jako test unitowy. Tak samo jak dla testu `UserService`. Byłby prostszy setup testu